mod/carrierwave/set/type/file.rb

```diff
@@ -9,7 +9,8 @@ def select_action_by_params params
 
   def last_content_action_id
     # find action id from content (saves lookups)
-    db_content.to_s.split(%r{[/.]})[-2]
+    aid = db_content.to_s.split(%r{[/.]})[-2]
+    aid.number? ? aid.to_i : nil
```

Comment:
this was at least temporarily a codename when changing from coded to local storage.